.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.0.0
+ruby-3.0.1
```

Comment:
Bumping the Ruby version was necessary to fix some tests incorrectly erroring locally.